clients/privacy-center/packages/fides-consent/src/fides-consent.ts

```diff
@@ -25,9 +26,19 @@ const defaults = makeDefaults({
  */
 const consent = getConsentCookie(defaults);
 
+/**
+ * Create a wrapped `banner` function that injects the default consent settings
+ * as an argument, so it can be used as Fides.banner()
+ */
+const banner = async (
+  extraOptions?: ConsentBannerOptions
+): Promise<void> => initBanner.call(undefined, defaults, extraOptions);
+
 const Fides = {
   consent,
 
+  banner,
+  getBannerOptions,
```

Comment:
This getter doesn't match the DX of the other functions on `Fides` object which felt a bit weird?